src/Microsoft.ML.Core/Utilities/ResourceManagerUtils.cs

```diff
@@ -269,12 +253,19 @@ private Exception DownloadResource(IHostEnvironment env, IChannel ch, WebClient
             string tempPath = Path.GetFullPath(Path.Combine(Path.GetDirectoryName(path), "temp-resource-" + guid.ToString()));
             try
             {
+                Console.WriteLine(String.Format("Trying to view file through the url: {0}", uri.ToString()));
```

Comment:
Unneeded; same info is printed in calling function on line: `ch.Info($"Downloading {fileName} from {url} to {filePath}");`
```suggestion
```